packages/runtime-core/src/h.ts

```diff
@@ -201,25 +202,35 @@ export function h<P>(
 
 // Actual implementation
 export function h(type: any, propsOrChildren?: any, children?: any): VNode {
+  // #6913 disable tracking block in h function
+  const doCreateVNode = (type: any, props?: any, children?: any) => {
```

Comment:
Just stumbled upon this as I was reading the changelog. Is there a reason why every call to `h` will now (afaik) create an inline version of `doCreateVNode`? Can't this be created outside of the scope of calling `h()`?

Reply:
You are right,  I made some simplifications to the `h` function and removed the `doCreateVNode` function. See https://github.com/vuejs/core/pull/13841